components/domains/externalDomainActions.tsx

```diff
@@ -20,6 +23,8 @@ const ExternalDomainActions: FunctionComponent<ExternalDomainActionsProps> = ({
 }) => {
   const { address } = useAccount();
   const [isOwner, setIsOwner] = useState<boolean>(false);
+  const [isTransferFormOpen, setIsTransferFormOpen] = useState<boolean>(false);
+  const domainKind = getDomainKind(domain as string);
```

Comment:
Why do you need to onforce `as string` ?

Reply:
no need actually